generators/server/index.js

```diff
@@ -154,6 +154,11 @@ module.exports = JhipsterServerGenerator.extend({
                 this.messageBroker = false;
             }
 
+            this.apiFirst = this.config.get('apiFirst') === 'no' ? false : this.config.get('apiFirst');
```

Comment:
Even if extensible in future its still more manageable to have one flag per option as it keeps ejs more readable and easy to manage. I wanted to change existing options this way and already did that for test frameworks